app/assets/stylesheets/application.scss

```diff
@@ -17,6 +17,17 @@
 
  */
 
+//top-left logo
+ .top-bar .name h4 a:before {
```

Comment:
Line should be indented 0 spaces, but was indented 1 spaces<br>Selector should have depth of applicability no greater than 2, but was 4